android/app/src/androidTest/java/com/gooddollar/DetoxTest.java

```diff
@@ -1,4 +1,4 @@
-package org.gooddollar;
+package com.gooddollar;
```

Comment:
org

Reply:
I know, but to keep it as org I need to change the appId everywhere in the android project. will tackle that later